torch/_dynamo/convert_frame.py

```diff
@@ -349,9 +350,9 @@ def exception_handler(
 
 
 FRAME_COUNTER = 0
-FRAME_COMPILE_COUNTER: typing.Counter[
-    Union[int, FrameStateSizeEntry]
-] = collections.Counter()
+FRAME_COMPILE_COUNTER: typing.Counter[Union[int, FrameStateSizeEntry]] = (
+    collections.Counter()
+)
```

Comment:
fbcode formatter causing thrash with the oss formatter. you may fail lint this way

Reply:
Thanks for pointing it out! Let me revert them.